pyproject.toml

```diff
@@ -3,6 +3,7 @@ name = "sfdo-template-helpers"
 version = "0.4.0"
 description = "A set of Django helpers and utils used by sfdo-template projects."
 authors = [
+    "David Glick <dglick@salesforce.com>",
```

Comment:
Not intending to claim primary credit here, but poetry-setup takes the setup.py author from the first entry here, and I think it makes sense for it to be a current employee of SFDO.